13_1_Verifying_Your_Lightning_Setup.md

```diff
@@ -0,0 +1,104 @@
+```
+cat >> ~/.bash_profile <<EOF
+alias lndir="cd ~/.lightning/" #linux default c-lightning path
+alias lnc="lightning-cli"
+alias lnd="lightningd"
+alias lninfo='lightning-cli getinfo'
+EOF
+```
+After you enter these aliases you can either `source .bash_profile` to input them or just log out and back in.
+
+Note that these aliases includes shortcuts for running `lightning-cli`, for running `lightningd`, and for going to the c-lightning directory. These aliases are mainly meant to make your life easier. We suggest you create other aliases to ease your use of frequent commands (and arguments) and to minimize errors. Aliases of this sort can be even more useful if you have a complex setup where you regularly run commands associated with Mainnet, with Testnet, _and_ with Regtest, as explained further below.
+
+With that said, use of these aliases in _this_ document might accidentally obscure the core lessons being taught about c-lightning, so the only alias directly used here is `lninfo` because it encapsulatea  much longer and more complex command. Otherwise, we show the full commands; adjust for your own use as appropriate.
+
+## Run lightningd
+
+You'll begin your exploration of the Lightning network with the `lightning-cli` command. However, lightningd _must_ be running to use lightning-cli, as lightning-cli sends JSON-RPC commands to the lightningd. If you used our standard setup, lightningd should already be up and running. You can double check by looking at the process table.
+```
+$ ps auxww | grep lightningd
+standup   7817  1.1  0.2  95280  8964 pts/18   D+   14:13   0:00 lightningd --network=testnet
+```
+If it's not running, you'll want to run `lightningd --network=testnet` by hand.
+
+## Verify your node
+
+You should have an output like this indicating your node is ready if blockheight  shows a height value that match with your most recent number getblockcount `bitcoin-cli getblockcount`  command output.
+
+```
+$ lightning-cli --network=testnet getinfo
+{
+   "id": "03fce2a20393a65b9d6cab5425f4cd33ddc621ade458efd69d652917e2b5eaf59c",
+   "alias": "VIOLENTIRON",
+   "color": "03fce2",
+   "num_peers": 0,
+   "num_pending_channels": 0,
+   "num_active_channels": 0,
+   "num_inactive_channels": 0,
+   "address": [],
+   "binding": [
+      {
+         "type": "ipv6",
+         "address": "::",
+         "port": 9735
+      },
+      {
+         "type": "ipv4",
+         "address": "0.0.0.0",
+         "port": 9735
+      }
+   ],
+   "version": "v0.8.2-398-g869fa08",
+   "blockheight": 54959,
+   "network": "testnet",
+   "msatoshi_fees_collected": 0,
+   "fees_collected_msat": "0msat",
+   "lightning-dir": "/home/user/.lightning/testnet"
+}
+```
+
+If node is still sycing with bitcoin network you should see a message like this.
+
+```
+"warning_bitcoind_sync": "Bitcoind is not up-to-date with network."
+```
+If you're not up-to-date, you'll get a message to you `getinfo command` like this:
```

Comment:
Really it's local database c-lightning, but we could call it the lightning daemon in my opinion.

It seems confusing but if we can differentiate the commands as you have proposed I think it adds more value to have the two implementations together since it allows to exemplify the bitcoin transfer on each side. Additionally at BitcoinStandUp we are giving the option to choose any alternative even if the main implementation is c-lightning.

1.) I can add a specific section showing how to install LND.
2-3) I think your proposal c$ and lnd$ is awesome.
4) My intention is to use the LND commands to the minimum possible but if a few times to show the status of the channel on the other side and how money transfers are received.